docs/_coverpage.md

```diff
@@ -1,13 +1,14 @@
-![logo](cloud.png)
+![logo](aresx_logo.png)
 
-# ARESx <small>2021</small>
+# ARESx <small>2020-2021</small>
 
-> Official webpage for the ARESx team
+> Official webpage for the ARESx CTF team
 
-- Some fancy description to fill the space
-- could go here
-- you know what I mean!
+- We participate in CTFs on a weekly basis
+- Our members are from all over the world
+- We're currently in the top 25 on CTF Time!
```

Comment:
I believe it should be spelled as CTFtime with no space

Reply:
Done, thanks!